src/components/Contacts/ContactListTable.jsx

```diff
@@ -42,110 +27,36 @@ const CustomToolbar = () => (
 const ContactListTable = ({ contacts, deleteContact }) => {
   const [showMessageModal, setShowMessageModal] = useState(false);
   const [messageToField, setMessageToField] = useState('');
+  const theme = useTheme();
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
 
   const handleSendMessage = (contactId) => {
     setShowMessageModal(!showMessageModal);
-    setMessageToField(contactId.value.podUrl);
+    setMessageToField(isSmallScreen ? contactId.podUrl : contactId.value.podUrl);
   };
```

Comment:
I think the state to set would not need to change,  just the size of the input fields used needs to be corrected?

Reply:
Could you elaborate please?

Reply:
Does contactId change when the screen size changes?

Reply:
For whatever reason, this is what happens when using `contactId.value.podUrl` on mobile:

<img width="945" alt="2024-06-26 (8)" src="https://github.com/codeforpdx/PASS/assets/83156697/1a7acd54-8fe7-40e1-bc0f-e8a9bff3471b">

The "To" field does not populate and it renders this error.

Reply:
I think it supposed to just be `contactId.podUrl`. Use that as the main variable for the set function:

```
setMessageToField(contactId.podUrl);
```

Reply:
That has the opposite effect for me. It messes up sending a message on the desktop version

Reply:
I would check the state of the object being passed in and see why the changes occur when switching between mobile and desktop views.

Reply:
Just found the answer.

![Screenshot 2024-06-26 at 9 29 12 PM](https://github.com/codeforpdx/PASS/assets/14917816/f3e2513b-1eff-476c-8f56-d707bf98048f)

Since the object does change depending on what mode it's using, then this should be fine.